components/coding/profileV2/achievement_components/findBadgeDiff.ts

```diff
@@ -0,0 +1,54 @@
+import { IntroCourseUnit } from "../../../../graphql/coding/userBadges/fetchUserBadges";
+import { CodingBadge } from "../../../../graphql/coding/userBadges/fetchUserBadges";
+
+const findBadgeDiff = (
```

Comment:
this function has come a looooooong way from when we first started... great work!